src/v0.4/parseExpr_staged.jl

```diff
@@ -304,34 +316,26 @@ end
 
 @generated addToExpression_reorder(ex, arg) = :(addToExpression(ex, 1.0, arg))
 
-@generated function addToExpression_reorder(ex, args...)
-    if !isleaftype(ex) || mapreduce(t -> !isleaftype(t), |, args)
-        error("Can't process abstract types")
+@generated function addToExpression_reorder(ex, x, y)
+    if x <: Union(Variable,AffExpr) && y <: Number
+        x, y = y, x
```

Comment:
Does this do anything below? ``x`` and ``y`` are just used as symbols in the expression returned